### ISSUE TITLE
Accessibility Issue for Screen Reader Users

### ISSUE THREAD

**Issue by Deyce1988** [qTox/qTox#1215]
The QTox Windows Client is not accessible for Users with Assistive Technologies.
See: http://qt-project.org/doc/qt-4.8/accessible.html

**Comment by tyrylu** [comment]
Or consider upgrading to the latest QT5 version, there were some substantial accessibility improvements and the entire qtaccessibblewidgets stuff was simplified.

**Comment by sudden6** [comment]
duplicate of https://github.com/tux3/qTox/issues/300?
